font-noto-cjk.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: font-noto-cjk
+  version: 0_git20220127
+  epoch: 0
+  description: Google's font family that aims to support all the world's languages (cjk)
+  copyright:
+    - license: OFL-1.1
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - fontconfig
+      - git
+      - xz
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: fa2551276a47c4a8fa15b451318a3e8e3aedb9b99f4f55028bd024a1f026bd13
+      uri: https://dev.alpinelinux.org/archive/font-noto-cjk/font-noto-cjk-${{package.version}}.tar.xz
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/share/fonts/noto
+      _rev=Sans2.004
+      git clone --depth=1 -b $_rev https://github.com/notofonts/noto-cjk
+      cp ./noto-cjk/*/OTC/*.ttc .
```

Comment:
Could we `git-checkout` https://github.com/notofonts/noto-cjk rather than this fetch?